.cargo/config

```diff
@@ -6,5 +6,10 @@ rustflags = [
     "-C", "linker=gcc",
 ]
 
+[target.x86_64-apple-darwin]
```

Comment:
Is this needed? If so, needed for arm64?

Reply:
removed